backend/api/v2beta1/swagger/pipeline.upload.swagger.json

```diff
@@ -1,313 +1,246 @@
-        "type": "object",
-        "properties": {
-          "error": {
+        },
+        "parameters": [
+          {
+            "name": "uploadfile",
+            "in": "formData",
+            "required": true,
+            "type": "file",
+            "description": "The pipeline to upload. Maximum size of 32MB is supported."
+          },
+          {
+            "name": "name",
+            "in": "query",
+            "required": false,
             "type": "string"
           },
-          "code": {
-            "type": "integer",
-            "format": "int32"
+          {
+            "name": "pipelineid",
+            "in": "query",
+            "required": false,
+            "type": "string"
           },
-          "details": {
-            "type": "array",
-            "items": {
-              "$ref": "#/definitions/protobufAny"
-            }
-          }
-        }
-      },
-      "apiUrl": {
-        "type": "object",
-        "properties": {
-          "pipeline_url": {
+          {
+            "name": "description",
+            "in": "query",
+            "required": false,
             "type": "string"
-          }
-        }
-      },
-      "protobufAny": {
-        "type": "object",
-        "properties": {
-          "type_url": {
-            "type": "string",
-            "description": "A URL/resource name that uniquely identifies the type of the serialized\nprotocol buffer message. The last segment of the URL's path must represent\nthe fully qualified name of the type (as in\n`path/google.protobuf.Duration`). The name should be in a canonical form\n(e.g., leading \".\" is not accepted).\n\nIn practice, teams usually precompile into the binary all types that they\nexpect it to use in the context of Any. However, for URLs which use the\nscheme `http`, `https`, or no scheme, one can optionally set up a type\nserver that maps type URLs to message definitions as follows:\n\n* If no scheme is provided, `https` is assumed.\n* An HTTP GET on the URL must yield a [google.protobuf.Type][]\n  value in binary format, or produce an error.\n* Applications are allowed to cache lookup results based on the\n  URL, or have them precompiled into a binary to avoid any\n  lookup. Therefore, binary compatibility needs to be preserved\n  on changes to types. (Use versioned type names to manage\n  breaking changes.)\n\nNote: this functionality is not currently available in the official\nprotobuf release, and it is not used for type URLs beginning with\ntype.googleapis.com.\n\nSchemes other than `http`, `https` (or the empty scheme) might be\nused with implementation specific semantics."
           },
-          "value": {
-            "type": "string",
-            "format": "byte",
-            "description": "Must be a valid serialized protocol buffer of the above specified type."
+          {
+            "name": "namespace",
+            "in": "query",
+            "required": false,
+            "type": "string"
```

Comment:
I just realized that `PipelineVersion` is not directly namespaced, but rather inherits it from the parent Pipeline. Backend indeed does not use the `namespace` from the `request`: https://github.com/kubeflow/pipelines/blob/master/backend/src/apiserver/server/pipeline_upload_server.go#L232